src/rp2_common/hardware_gpio/include/hardware/gpio.h

```diff
@@ -365,46 +375,243 @@ enum gpio_drive_strength gpio_get_drive_strength(uint gpio);
  *   1 | High level
  *   2 | Edge low
  *   3 | Edge high
+ *
+ *  Which are specified in \ref gpio_irq_level
  */
 void gpio_set_irq_enabled(uint gpio, uint32_t events, bool enabled);
 
-/*! \brief Enable interrupts for specified GPIO
+#ifndef GPIO_CALLBACK_IRQ_ORDER_PRIORITY
```

Comment:
Should there be a PICO_CONFIG stanza for this, or is it something that you don't expect users to change?